src/form.php

```diff
@@ -62,10 +70,56 @@
 -->
                         <label for="reply-to">Reply-to</label>
                         <input id="reply-to" name="reply-to" class="form-control form-control-lg bg-dark text-white" placeholder="for example: tapahtumavastaava@blanko.fi" required="" />
-
+                        <label for="tags">Tag</label> <br/>
```

Comment:
These could be sorted alphabetically (manually is fine, frontend functionality not necessary)